client/src/pages/Register/index.tsx

```diff
@@ -30,6 +30,7 @@ export interface FormDataState {
   formData: FormData;
   totals: PricingResults;
   step: "registration" | "payment";
+  registrationId?: string | number;
```

Comment:
`registrationId` should be replaced with `registrationUUID` everywhere to match the POST request/response parameter